dbfit-java/build.gradle

```diff
@@ -290,10 +290,71 @@ project('teradata') {
 project('netezza') {
     description = 'DBFit netezza engine'
 
+    def propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+    def jdbcDriverClass = 'org.netezza.Driver'
+    def jdbcUrlPrefix = 'jdbc:netezza://'
+    def jdbcUrl
```

Comment:
I don't   have a precise answer. One thought is to have a function which returns all details.